### DISCUSSION TITLE
[PYFLUENT] Launch sbatch job

### DISCUSSION THREAD

**Discussion by jeremy-jimmy** [ansys/pyfluent#1475] [Q&A] [answered]
Hi,

I'm facing difficulties to launch a job on my cluster using "pyfluent" python package, the example given in the documentation doesn't give enought details to solve my issues.

My slurm job is as follow : 
```
#!/bin/bash
#SBATCH --job-name=FLUENT_ALONE
#SBATCH --nodes=1  # nodes requested
#SBATCH --ntasks=1  # tasks requested
#SBATCH -o OUTPUT_SLURM.txt
#SBATCH --cpus-per-task=4 #nb cpu per tasks
#SBATCH --partition=cfd # select the queue
#SBATCH --export=ALL

export ANSYSLI_SERVERS=2325@XXXXXXXX
export SERVER=1055@XXXXXXXX
export ANSYSLMD_LICENSE_FILE=1055@XXXXXXXX

source .VENV_PYTHON_310/bin/activate
NODEFILE="$SLURM_SUBMIT_DIR/slurmhosts.$SLURM_JOB_ID.txt"
scontrol show hostname $SLURM_NODELIST >$NODEFILE

python FLUENT_DEBUG.py $NODEFILE
```
and my python script is as follow : 
```
import os
import sys
import ansys.fluent.core as pyfluent
os.environ["PYFLUENT_FLUENT_ROOT"] = "/fsx/ansys_inc/v222/fluent/" 
pyfluent.set_log_level("DEBUG")

print("NODEFILE", sys.argv[1])
nodefile = sys.argv[1]
solver = pyfluent.launch_fluent(precision="double", processor_count=4, mode="solver", show_gui=False, start_timeout = 20, product_version="22.2.0" ,additional_arguments=f'-g -r22.2.0 -ssh -mpi=intel -cnf={nodefile}')

print(solver.check_health())
```

==> my fluent solver is not able to launch.

it's probably a beginner mistake but I can't see it.

**Comment by seanpearsonuk** [comment]
@rploconnor @PyPablo is there anyone on your side who can provide guidance here?

**Reply by jeremy-jimmy** [reply, marked as answer]
I managed to solve my issue as mention it was beginner mistake due to misconfiguration of the communication to the license server.

**Comment by dnwillia-work** [comment]
@jeremy-jimmy For what it's worth you don't need the NODEFILE variable or scontrol calls in your shell script, nor do you need to pass the node file in your Python script (though it will work as you coded it):

https://fluent.docs.pyansys.com/version/stable/user_guide/launching_ansys_fluent.html#scheduler-support

pyfluent does it for you if you want.
